### DISCUSSION TITLE
Why the httpclient5 and httpcore5 libraries are present in DITA-OT 4.1?

### DISCUSSION THREAD

**Discussion by jlacour31** [dita-ot/dita-ot#4232] [Q&A] [answered]
In DITA-OT 4.1 distribution there are 3 extra libraries:
- httpclient5-5.1.3.jar
- httpcore5-5.1.3.jar
- httpcore5-h2-5.1.3.jar

These libraries are brought into the distribution by xmlresolver-5.1.2.jar used in Saxon.
I think they can be excluded.

**Comment by jelovirt** [comment, marked as answer]
If only xmlresolver.org uses them and still works in all normal uses case without, we should remove them as bloat.

**Comment by jlacour31** [comment]
I created https://github.com/dita-ot/dita-ot/issues/4235 for removal.
